src/content/docs/style-guide/images/create-videos.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Create videos
+tags:
+  - Tech writer style guide
+  - Processes and procedures
+redirects:
+---
+
+There are many ways to create videos, but here are some tips that will likely apply to a lot of video production projects. 
+
+## Before you start [#before-you-start]
+
+Before you start, talk to the Nerd Bytes team and make sure they're not working on a similar video. 
```

Comment:
@zuluecho9 I think you could drop the "Before you start" since that's already in the heading.